internal/moreos/proc_windows.go

```diff
@@ -69,7 +69,7 @@ func ensureProcessDone(p *os.Process) error {
 		}
 		return os.NewSyscallError("OpenProcess", e)
 	}
-	defer syscall.CloseHandle(h)
+	defer syscall.CloseHandle(h) //nolint
```

Comment:
oh which kind of lint errors we got with this?

Reply:
good point :D It is the usual defer thing where we don't have a nested function to check the thing that closes the thing didn't also err. Pretty ok with ack'ing, but agree it should be precise!

```suggestion
	defer syscall.CloseHandle(h) //nolint:errcheck
```